frontend/src/App.tsx

```diff
@@ -1,21 +1,53 @@
 import { CssVarsProvider } from "@mui/joy/styles";
 import CssBaseline from "@mui/joy/CssBaseline";
-import React from "react";
+import React, { ReactNode } from "react";
 import GameReportForm from "./GameReportForm";
 import GameReports from "./GameReports";
 import Rankings from "./Rankings";
+import Box from "@mui/joy/Box";
 import IconButton from "@mui/joy/IconButton";
+import DownloadIcon from "@mui/icons-material/Download";
 import Drawer from "@mui/joy/Drawer";
+import MuiLink from "@mui/joy/Link";
 import List from "@mui/joy/List";
 import ListItemButton from "@mui/joy/ListItemButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+import Typography from "@mui/joy/Typography";
 import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
 
 export default function App() {
     return (
         <CssVarsProvider>
             <CssBaseline />
             <BrowserRouter>
-                <DrawerNavigation />
+                <header
+                    style={{
+                        display: "flex",
+                        alignItems: "center",
+                        width: "100%",
+                        position: "sticky",
+                        top: 0,
+                        zIndex: 1000,
+                        margin: "0 0 20px 0",
+                        padding: "10px 10px",
+                        boxShadow: "0 0 2px 0 rgba(0, 0, 0, 0.5)",
+                        color: "white",
+                        backgroundColor: "var(--joy-palette-primary-solidBg)",
+                    }}
+                >
+                    <DrawerNavigation />
+                    <Typography
+                        level="title-lg"
+                        sx={{
+                            color: "white",
+                            width: "100%",
+                            textAlign: "center",
+                        }}
+                    >
+                        WotR Community Ladder
```

Comment:
Wasn't sure about this, maybe it should be just `WotR Community`?

Reply:
Idk! This is good for now I think?